src/patches/ApplySimGameEventResult.cs

```diff
@@ -6,6 +6,7 @@
 using BattleTech;
 
 namespace WarTechIIC {
+
     [HarmonyPatch(typeof(SimGameState), "ApplySimGameEventResult", new Type[] {typeof(SimGameEventResult), typeof(List<object>), typeof(SimGameEventTracker)})]
     public static class SimGameState_ApplySimGameEventResult_Patch {
         // WIIC_give_Sol_to_ClanWolf
```

Comment:
Also need to update the sample tags in the comments here, and update the examples in the README.md.